app/models/solution.rb

```diff
@@ -13,4 +13,16 @@ def initialize(entry)
     @url = entry.fields[:url]
     @category = entry.fields[:category]
   end
+
+  def self.find_by_slug(slug)
+    entry = ContentfulClient.entries(
+      content_type: "solution",
+      'fields.slug': slug,
+      include: 1,
+      select: "sys.id,fields.title,fields.description,fields.summary,fields.slug, fields.provider_name, fields.url"
```

Comment:
The use of spaces and commas is inconsistent. That probably doesn't matter to Contentful, but I prefer consistency!

I wonder if it would be more readable if it were something like this:

```
%w[
  sys.id
  fields.title
  fields.description
  # ...
  fields.url
].join(",")
```